resources/assets/js/app.js

```diff
@@ -1,22 +1,68 @@
-
 /**
  * First we will load all of this project's JavaScript dependencies which
  * includes Vue and other libraries. It is a great starting point when
  * building robust, powerful web applications using Vue and Laravel.
  */
 
-require('./bootstrap');
+require("./bootstrap");
```

Comment:
Reminder to the whole team! @Tokidaze  @Oleg-Lopes  @SebastianGerS 
Remove bootstrap (css-library) and Vue as dependencies - since we doesn't use them.